modules/test/playwright/pages/document-library-web/documentLibrary.page.ts

```diff
@@ -37,9 +47,43 @@ export class DocumentLibraryPage {
 		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
 	}
 
+	async openNewButton() {
+		await this.page.getByRole('button', {name: 'New'}).click();
+	}
+
+	async openCreateAIImage() {
+		await this.openNewButton();
+
+		await this.page
+			.getByRole('menuitem', {
+				name: 'Create AI Image',
+			})
+			.click();
+	}
+
 	async openOptionsMenu() {
 		await this.optionsMenu
 			.and(this.page.locator('[aria-haspopup]'))
 			.click();
 	}
+
+	async enableAICreator() {
+		await this.aiCreatorPageSettingsPage.enableDalleCreateImages();
+	}
+
+	async disableAICreator() {
+		await this.aiCreatorPageSettingsPage.disableDalleCreateImages();
+	}
+
+	async addApiKey() {
+		await this.aiCreatorPageSettingsPage.addApiKey();
+	}
+
+	async removeApiKey() {
+		await this.aiCreatorPageSettingsPage.removeApiKey();
+	}
+
+	async addGogoShellCommand(command: string) {
+		await this.gogoShellPage.addCommand(command);
+	}
```

Comment:
I prefer to avoid these proxy patterns and use the page directly (aiCreatorPageSettingsPage/gogoShellPage)